foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/NonHelmKubernetesGenerator.java

```diff
@@ -15,10 +15,9 @@
 public class NonHelmKubernetesGenerator extends AbstractKubernetesGenerator {
     /*--~-~-~~
      * Usages:
-     * | Target              | Template                                                        | Generated File                           |
-     * |---------------------|-----------------------------------------------------------------|------------------------------------------|
-     * | trainingNoOpFile    | deployment/model-training-job/model-training-job.no-op.yaml.vm  | apps/${appName}/${appName}.yaml          |
-     * | sparkWorkerJobFile  | deployment/spark-worker-image/spark-worker-image.yaml.vm        | apps/${appName}/spark-worker-image.yaml  |
```

Comment:
Would also add to the Release Notes that the `spark-worker-image.yaml` can be deleted.